README.md

```diff
@@ -188,9 +188,198 @@ The procedure is described in `experiments/splash/plot.py`.
 Internally, we use matplotlib, but you're free to use anything else in your experiments - just re-write `plot.py`.
 
 # Example 2: Modifying an existing experiment
+In this example we will modify the splash experiment.
 
-TBD
+In the first part of this example we will:
+ - Change the evaluation from cycles to peak memory consumption.
+ - Change the graph to display absolute numbers.
```

Comment:
```suggestion
 - Change the plot to display absolute numbers.
```

In computer science, the word graph has a very specific meaning: https://en.wikipedia.org/wiki/Graph_theory

If you use it as a synonym to "plot", people will get confused.